hops/serving.py

```diff
@@ -312,6 +318,13 @@ def _create_or_update_serving_rest(model_path, model_name, serving_type, model_v
                                       "user msg: {}".format(resource_url, response.status_code, response.reason,
                                                             error_code, error_msg, user_msg))
 
+
+def _detect_model_server(artifact_path):
```

Comment:
Might be nice to print what model_server the method infers it to be if you think it's useful.